so_vector/index.json

```diff
@@ -22,6 +22,7 @@
       },
       "title": {
         "type": "text"
+        {% if source_mode == "synthetic" %},"store": true{% endif %}
```

Comment:
Does it mean that synthetic source is not supported for text fields?

Reply:
It *is*, but only if you `store` them or have a `keyword` sub field. Depending on your goals with the `text` field you might want to do one or the other and I didn't want to assume for anyone.